templates/base.html

```diff
@@ -124,13 +124,20 @@ <h1>{% block title %}{{product_name | capitalize }} {{ os_family | capitalize}}
             <p>
               You will need to explicitly install a Java runtime environment, because Oracle's Java RPMs are incorrect and fail to register as providing a java dependency.
               Thus, adding an explicit dependency requirement on Java would force installation of the OpenJDK JVM.
-            <p>
-
-            <ul>
-              <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
-              <li>2.54 (2017-04) and newer: Java 8</li>
-              <li>1.612 (2015-05) and newer: Java 7</li>
-            </ul>
+      <p>
+          <b> LTS Version</b>
+              <ul>
+                    <li>2.346 (2022-06) and newer: Java 8 or Java 11</li>
```

Comment:
The upcoming LTS version supports running on Java 17 too

Reply:
thanks @NotMyFault, will update accordingly!